src/Entity/Monei2Payment.php

```
@@ -561,4 +561,39 @@ public function toArrayLegacy()
             'date_upd' => $this->getDateUpdFormatted(),
         ];
     }
+
+    /**
+     * Find multiple entities by criteria
+     * 
+     * @param array $criteria Associative array of field => value pairs
+     * @param string $orderBy Order by clause (ignored for simplicity, always uses date_add DESC)
+     * @return array Array of Monei2Payment objects
+     */
+    public static function findBy($criteria, $orderBy = null)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$orderBy'. (undefined)

(UnusedFormalParameter)